### PR TITLE
Account tests - minor refactor

### PR DESCRIPTION
**Pull request by ilayaraja89**

### CODE REVIEW

**Comment by ilayaraja89** [comment]
@sudhirj Made minor refactor in this 2f0eb11. Please review and merge this, to use the master as baseline for transactions API.

File: controllers/accounts_test.go [hunk 1]
```diff
@@ -0,0 +1,68 @@
...
+	rr := httptest.NewRecorder()
+
+	handler := middlewares.ContextMiddleware(GetAccountInfo, as.context)
+	req, err := http.NewRequest("GET", ACCOUNTS_INFO_API+"?id=100", nil)
+	if err != nil {
+		t.Fatal(err)
+	}
+	handler.ServeHTTP(rr, req)
+
+	account := models.Account{}
```
Comment (anuradhag) [inline controllers/accounts_test.go]: Can initialize this after asserting response code

File: models/accounts_test.go [hunk 2]
```diff
@@ -0,0 +1,44 @@
...
+		as.db = db
+	}
+}
+
+func (as *AccountsSuite) TestAccountsInfoAPI() {
+	t := as.T()
+
+	accountsDB := AccountDB{DB: as.db}
+	account := accountsDB.GetByID("100")
+	// test valid id
```
Comment (anuradhag) [inline models/accounts_test.go]: We dont need comments here as the assertions are self explanatory.

File: controllers/accounts_test.go (hunk 1 above)
Comment (ilayaraja89) [inline controllers/accounts_test.go]: Done

File: models/accounts_test.go (hunk 2 above)
Comment (ilayaraja89) [inline models/accounts_test.go]: removed unnecessary comments

**Pull request merged**